server.js

```diff
@@ -29,7 +30,60 @@ app.post('/api/v1/diary', (request, response) => {
   } else {
     response.status(201).json({name})
   }
+})
+
+app.get('/api/v1/foods/:id', (request, response) => {
+  let id = request.params.id
+  Food.find(id)
+  .then( (data) => {
+    if (data.rowCount === 0) { return response.sendStatus(404) }
+
+    response.json(data.rows[0])
+  })
+})
 
+app.get('/api/v1/foods', (request, response) => {
+  Food.all()
+  .then( (data) => {
+    if (data.rowCount === 0) { return response.sendStatus(404) }
```

Comment:
i changed line 48 to `if (data.rowCount === 0) { return data.rows }` but it's getting hung up in postman at that line. Any ideas why?